src/config.test.ts

```diff
@@ -148,4 +148,76 @@ describe("config", () => {
     const { config } = await import("./config.js");
     expect(config.options.scraping.daysBack).toBe(15);
   });
+
+  it.each([
```

Comment:
I don't like the duplication for the 2 storages, consider having one urls array and 2 it.each one for each storage